helm/prometheus-rules/templates/platform/atlas/alerting-rules/grafana.rules.yml

```diff
@@ -47,3 +47,15 @@ spec:
         severity: page
         team: atlas
         topic: observability
+    - alert: GrafanaPostgresqlRecoveryTestFailed
+      annotations:
+        description: '{{`grafana-postgresql db automated recovery test failed with cluster {{ $labels.cluster }}.`}}'
+        runbook_url: https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/grafana-down/
+      expr: sum(cnpg_collector_up{namespace="monitoring", cluster="grafana-postgresql-recovery-test"}) by (cluster, cluster_id, installation, pipeline, provider) >= 1
```

Comment:
Maybe add a comment about this `cnpg_collector_up` metrics? What does it mean, and what its values mean?

Reply:
> So, basically, if you have 2 or more grafana-postgresql-recovery-test pods in the same CNPG cluster, it means there was a problem with the recovery test?

No, if you have 1 or more grafana-postgresql-recovery-test pods that have been around for more than 1 hour, this means tha there's a problem as the test timeout has been reached for a while and the pods should be gone.